tests/bootstrap.php

```diff
@@ -5,3 +5,10 @@
 require_once __DIR__ . '/../vendor/autoload.php';
 
 PHPStanTestCase::getContainer();
+
+if (
+	!is_file(__DIR__ . '/../vendor/doctrine/orm/src/Mapping/Driver/AnnotationDriver.php')
+	&& !is_file(__DIR__ . '/../vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php')
+) {
+	require_once __DIR__ . '/../compatibility/AnnotationDriver.php';
```

Comment:
This is a nice idea indeed in order to have less code to update currently.

But I feel like it could still be useful in long term (certainly in another PR which can be merged before or after the doctrine bump) to add Attributes everywhere Annotation is used in tests in order to tests them too. Also, rather having some time `data/` and `data-attributes` folders with duplicated tests, a unique `data/` folder can be used with both annotation and attribute in the same class without any issue.

This way the logic
```
if (! class_exists(AnnotationDriver::class) {
    // use AttributeReader
} else {
   // use AnnotationReader (or decide based on PHP version if wanted)
}
```
can be use in the entity-manager.php.

Reply:
> to add Attributes everywhere Annotation

I disagree with this approach. I consider existing tests to be immutable. By changing them we might start testing different things by accident than originally intended.